addStyles.js

```diff
@@ -143,6 +143,20 @@ function createLinkElement(options) {
 function addStyle(obj, options) {
 	var styleElement, update, remove;
 
+	// If a transformCssOnLoad function was defined, run it on the css
+	if (options.transformCssOnLoad && obj.css) {
```

Comment:
@brafdlog Yep the sole `transform` is to jaded, personally in favour of `transformOnLoad`, the more precise and short the better 😛 . I will quit bikesheding about naming from this point, choose based on majority and preference :)